plone-and-python-community.rst

```diff
@@ -1231,31 +1231,32 @@ It is not the |br| Release Manager
 
     **Sprints**
 
-    more than one sprint per month in average
+    more than one sprint per month on average
 
-    **Symposiums**
+    **Symposia**
 
     specific subgroups or areas
 
-    * Plone Symposium East / Midwest (US University focus)
+    * Plone Symposium East / Midwest (US university focus)
```

Comment:
You could use this logo from PSM :) http://2014.plonesymp.org/++theme++plone-symposium-theme-20140313-clickabletitleandlogo/img/psmLogo.png